package.json

```diff
@@ -18,7 +18,6 @@
     "react": "18.2.0",
     "react-dom": "18.2.0",
     "react-icons": "^4.4.0",
-    "react-syntax-highlighter": "^15.5.0",
```

Comment:
We don't use this one anymore?

Reply:
It was being very problematic so I've removed it for now and I might have found a more accessible version, just need to look into it some more. So for now just black and white code blocks.